haskell/experimental/private/module.bzl

```
@@ -46,12 +46,20 @@ def haskell_module_impl(ctx):
 
     # Determine outputs
     with_profiling = is_profiling_enabled(hs)
+    hs_boot = paths.split_extension(src.path)[1] == ".hs-boot"
```

```suggestion
    hs_boot = paths.split_extension(src.path)[1] in [".hs-boot", ".lhs-boot"]
```
Following the above.

---

Fixed this too